geomesa-kafka/geomesa-kafka-datastore/src/main/scala/org/locationtech/geomesa/kafka/utils/GeoMessageSerializer.scala

```diff
@@ -111,7 +111,7 @@ object GeoMessageSerializer {
       try {
         // use the feature id if available, otherwise (for clear) use random shard
         if (keyBytes.length > 0) {
-          Math.abs(MurmurHash3.bytesHash(keyBytes)) % count
+          Math.abs(MurmurHash3.bytesHash(keyBytes, MurmurHash3.arraySeed)) % count
```

Comment:
Hmm... why did this change?  (Mainly curious)

Reply:
scala changed the implementation of this bytes hash in 2.11.10-ish... which would mean that hashes don't match any more, potentially causing duplicated data. calling it this way mimics the old behavior so keeps the hashes consistent across scala versions.

Reply:
this only turned up due to a random failing unit test, and was loads of fun to track down :P

Reply:
Good job!  Better us tracking it down in a unit test now....;)